scripts/monolith/install-dockolith.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+create_log_group "Install Dockolith"
+if [[ -z ${DOCKOLITH_BRANCH} ]]; then
+  export DOCKOLITH_BRANCH=master
+fi
+
+pushd ./scripts
+  rm -rf dockolith
+  echo "Cloning dockolith from branch: ${DOCKOLITH_BRANCH}"
+  git clone --depth 1 -b $DOCKOLITH_BRANCH https://github.com/okta/dockolith.git
+popd
+
+# Yarn "add" always modifies package.json https://github.com/yarnpkg/yarn/issues/1743
+# Make a backup of package.json and restore it after install
+cp package.json package.json.bak
+yarn add -DW --no-lockfile file:./scripts/dockolith
```

Comment:
I think this is where the build happens. Can add --ignore-scripts option to skip the build step

Reply:
Adding `--ignore-scripts` here prevents dockolith from building